src/main/java/ua/ithillel/lesson6/MyUtils.java

```diff
@@ -0,0 +1,106 @@
+package ua.ithillel.lesson6;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyUtils {
+
+    /*
+      1. Створити метод countOccurance, що приймає на вхід рядковий список як параметр і довільний рядок. Список
+      заповнений довільними словами 10-20 штук, які можуть повторюватись у необмеженій кількості. Порахувати скільки
+      разів зустрічається переданий рядок як другий аргумент.
+    */
+    public int countOccurrence(List<String> list, String item){
+        int counter = 0;
+        for (String elem : list) {
+            if(elem.equals(item)){
+                ++counter;
+            }
+        }
+
+        return counter;
+    }
+
+    /*
+      2. Створити метод toList, що приймає на вхід цілісний масив довільної довжини. Конвертувати масив у список
+      відповідного типу даних та повернути з методу.
+    */
+    public List<Integer> toList(int[] array){
+        List<Integer> list = new ArrayList<>();
+
+        for (int elem : array) {
+            list.add(elem);
+        }
+        return list;
+    }
+
+    /*
+      3. Створити метод findUnique, що приймає на вхід числовий список, що складається з довільних значень, які можуть
+      повторюватися в необмеженій кількості. Необхідно повернути новий числовий список, що містить тільки унікальні числа.
+      (Використовуючи лише List)
+    */
+    // --- берем в верхнем цикле 1-й объект списка, далее во вложенном цикле пробежимся по всему списку и при совпадении
+    // --- объекта удаляем его из текущего списка
+    public List<Integer> findUnique(List<Integer> list){
+
+        for (int i = 0; i < list.size(); i++) {
```

Comment:
```suggestion
        for (Integer iElem : list) {
```
те саме, але коротше